src/views/partials/header.hbs

```diff
@@ -3,6 +3,6 @@
       <meta name="viewport" content="width=device-width,
       initial-scale=1.0">
       <meta http-equiv="X-UA-Compatible" content="ie=edge">
-      <link rel="stylesheet" href="/css/style.css">
+      <!-- <link rel="stylesheet" href="/css/style.css"> -->
```

Comment:
If this is not needed, remove it

Reply:
This will be needed once we start the CSS (which is what I'm working on now...)

Reply:
You can add it in when it's needed ;)